netmanager/src/views/components/DataDisplay/DeviceView/DeviceLogs.js

```diff
@@ -246,10 +246,23 @@ const AddLogForm = ({ deviceName, deviceLocation, toggleShow, loading, setLoadin
     evt.preventDefault();
     const extracted_tags = [];
     tags && tags.map((tag) => extracted_tags.push(tag.value));
+    const storedData = localStorage.getItem('currentUser'); 
+    if (!storedData) {
+      console.error("Error: No user data found in local storage");
+      return;
+    }
+
+    const parsedData = JSON.parse(storedData);
+
+
     const logData = {
       date: selectedDate.toISOString(),
       tags: extracted_tags,
-      description: description
+      description: description,
```

Comment:
thanks @danielmarv ....since you can't yet get the user details, isn't it best to just remove the implementation from the PR? cc: @Codebmk 

And just have the PR be about the addition of the recallType field?

Reply:
is it ok now